lisa/tools/kdump.py

```diff
@@ -194,6 +194,10 @@ class KdumpBase(Tool):
 
     dump_path = "/var/crash"
 
+    kdump_config = {
```

Comment:
KdumpBase is a tool, and it is shared across the test-cases running in parallel. So, if it gets modified in one test-case, its affecting all the other test-cases. So, the current implementation is wrong.

@squirrelsc  @LiliDeng Can you please point me towards some reference where I can maintain node / test-case specific config parameters?

Reply:
@0xba1a pass parameter in `kdump.enable_kdump_service()` https://github.com/microsoft/lisa/blob/a6361265df2c47fd44785c1288d956ad61e3d31f/microsoft/testsuites/kdump/kdumpcrash.py#L341